docs/en/latest/deployment-modes.md

```diff
@@ -111,23 +111,77 @@ This method is more suitable for two types of users:
 1. Kubernetes(k8s)：Declarative API that dynamically updates the routing rules with a full yaml configuration.
 2. Different configuration centers: There are many implementations of the configuration center, such as Consul, etc., using the full yaml file for intermediate conversion.
 
-The routing rules in the `conf/apisix.yaml` file are loaded into memory immediately after the APISIX node service starts. And every time interval (default 1 second), will try to detect whether the file content is updated, if there is an update, reload the rule.
+### Modes
 
-*Note*: Reloading and updating routing rules are all hot memory updates. There is no replacement of working processes, since it's a hot update.
+Now, we have two standalone running modes, file-driven and API-driven.
 
-Since the current Admin API is based on the etcd configuration center solution, enable Admin API is not allowed when the Standalone mode is enabled.
+1. The file-driven mode is the kind APISIX has always supported.
 
-Standalone mode can only be enabled when we set the role of APISIX as data plane. We set `deployment.role` to `data_plane` and `deployment.role_data_plane.config_provider` to `yaml`.
+    The routing rules in the `conf/apisix.yaml` file are loaded into memory immediately after the APISIX node service starts. At each interval (default: 1 second), APISIX checks for updates to the file. If changes are detected, it reloads the rules.
 
-Refer to the example below:
+    *Note*: Reloading and updating routing rules are all hot memory updates. There is no replacement of working processes, since it's a hot update.
 
-```yaml
-deployment:
-  role: data_plane
-  role_data_plane:
-    config_provider: yaml
-#END
-```
+    This requires us to set the APISIX role to data plane. That is, set `deployment.role` to `data_plane` and `deployment.role_data_plane.config_provider` to `yaml`.
```

Comment:
Why is indentation added in front of these paragraphs? I don't think that's necessary.

Reply:
These text blocks must be in an ordered list, not aligned from the beginning of the line. The use of indentation accomplishes exactly that.

Reply:
![image](https://github.com/user-attachments/assets/eed96ad7-50c7-43b9-9eff-44734a9663b5)